src/main/scala/cromwell/engine/VassalCallActor.scala

```diff
@@ -0,0 +1,50 @@
+package cromwell.engine
+
+import akka.actor.{Actor, Props}
+import akka.event.{Logging, LoggingReceive}
+import cromwell.binding.values.WdlValue
+import cromwell.binding.{Call, CallInputs, WorkflowDescriptor}
+import cromwell.engine.backend.Backend
+import cromwell.engine.workflow.WorkflowActor
+import cromwell.engine.workflow.WorkflowActor.CallFailed
+import cromwell.engine
+import cromwell.binding.WdlExpression.ScopedLookupFunction
+import cromwell.binding._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.language.postfixOps
+import scala.util.{Try, Failure, Success}
+
+
+object VassalCallActor {
+  sealed trait VassalCallActorMessage
+  case class Finished(outputs: CallOutputs) extends VassalCallActorMessage
+  case class Failed(regret: Throwable) extends VassalCallActorMessage
+
+  def props(backend: Backend, command: String, workflowDescriptor: WorkflowDescriptor, call: Call, callInputs: CallInputs, lookup: ScopedLookupFunction): Props =
+    Props(new VassalCallActor(backend, command, workflowDescriptor, call, callInputs, lookup))
+}
+
+/** Actor to manage the execution of a single call. */
+class VassalCallActor(backend: Backend, command: String, workflowDescriptor: WorkflowDescriptor, call: Call, callInputs: CallInputs, lookup: ScopedLookupFunction) extends Actor with CromwellActor {
+  private val log = Logging(context.system, classOf[VassalCallActor])
+  val tag = s"VassalCallActor [UUID(${workflowDescriptor.shortId}):${call.name}]"
+
+  log.info(s"$tag STARTING $command")
+  backend.executeCommand(command, workflowDescriptor, call, callInputs, lookup) match {
+    case Success(callOutputs) =>
+      log.info(s"$tag FINISHING $command")
+      context.parent ! VassalCallActor.Finished(callOutputs)
+    case Failure(e) =>
+      log.error(s"$tag FAILING $command")
+      context.parent ! VassalCallActor.Failed(e)
+  }
+
+  override def receive = LoggingReceive {
+    case badMessage =>
+      val diagnostic = s"$tag: unexpected message $badMessage."
+      log.error(diagnostic)
+      context.parent ! engine.workflow.WorkflowActor.CallFailed(call, diagnostic)
```

Comment:
Do we really want to fail the call if it receives a spurious message? 

It does seem like behavior we'd want to know about and admittedly just logging messages is unlikely to ever be spotted, so perhaps so. OTOH it's not actually affecting the call.